tests/javascript-config/unit/fixtures/checkin-schema.js

```diff
@@ -0,0 +1,256 @@
+								},
+							},
+							'additionalProperties': false,
+						},
+					},
+					'https://api.eventespresso.com/extra_meta': {
+						'description': 'Array of objects describing the link(s) for this relation resource.',
+						'type': 'array',
+						'readonly': true,
+						'items': {
+							'type': 'object',
+							'properties': {
+								'href': {
+									'type': 'string',
+									'description': 'The link to the resource for the Extra_Meta relation(s) to this entity',
+								},
+								'single': {
+									'type': 'boolean',
+									'description': 'Whether or not there is only a single Extra_Meta relation to this entity',
+								},
+							},
+							'additionalProperties': false,
+						},
+					},
+					'https://api.eventespresso.com/change_log': {
+						'description': 'Array of objects describing the link(s) for this relation resource.',
+						'type': 'array',
+						'readonly': true,
+						'items': {
+							'type': 'object',
+							'properties': {
+								'href': {
+									'type': 'string',
+									'description': 'The link to the resource for the Change_Log relation(s) to this entity',
+								},
+								'single': {
+									'type': 'boolean',
+									'description': 'Whether or not there is only a single Change_Log relation to this entity',
+								},
+							},
+							'additionalProperties': false,
+						},
+					},
+				},
+				'additionalProperties': false,
+			},
+			'_calculated_fields': {
+				'description': 'Available calculated fields for this model.  Fields are only present in the response if explicitly requested',
+				'type': 'object',
+				'properties': [],
+				'additionalProperties': false,
+				'readonly': true,
+				'_protected': {
+					'description': 'Array of property names whose values were replaced with their default (because they are related to a password-protected entity.)',
+					'type': 'array',
+					'items': {
+						'description': 'Each name corresponds to a property that is protected by password for this entity and has its default value returned in the response.',
+						'type': 'string',
+						'readonly': true,
+					},
+					'readonly': true,
+				},
+			},
+			'_protected': {
+				'description': 'Array of property names whose values were replaced with their default (because they are related to a password-protected entity.)',
+				'type': 'array',
+				'items': {
+					'description': 'Each name corresponds to a property that is protected by password for this entity and has its default value returned in the response.',
+					'type': 'string',
+					'readonly': true,
+				},
+				'readonly': true,
+			},
+			'CHK_ID': {
+				'description': 'Check-in ID',
+				'type': 'integer',
+				'readonly': true,
+				'default': 0,
+				'primary_key': true,
+			},
+			'REG_ID': {
+				'description': 'Registration Id',
+				'type': 'integer',
+				'readonly': false,
+				'default': 0,
+				'foreign_key': {
+					'description': 'This is a foreign key the points to the given models.',
```

Comment:
???

maybe:

> This is a foreign key that points to the given models

or better yet (imho)

> This is a foreign key that points to the Registration model

Reply:
This will require a separate issue/pull request as it came from a direct REST response (that I copied and pasted into this file for a fixture).  In other words, this is a grammar issue in the REST API code in core.